app/invoices/route.js

```diff
@@ -57,32 +59,32 @@ export default AbstractModuleRoute.extend(ModalHelper, PatientListRoute, {
     }
   },
 
-  subActions: function() {
+  subActions: Ember.computed(function() {
     let actions = [{
-      text: 'Billed',
+      text: this.get('i18n').t('billing.navigation.billed'),
```

Comment:
Just wondering if you can use this `t` alias right away here? What is the difference then in `t('abc')` and `this.get('i18n).t('abc') ?`

Reply:
I'm following the pattern discussed in this issue, I'm not exactly sure why it's required as I haven't used Ember before but intitally I was using t right away and encounted the error described here.
https://github.com/jamesarosen/ember-i18n/issues/352

Reply:
Ok, just good to know, thx

Reply:
Just as an FYI, I think it's mostly to do with `this` and the context of the array inside of an es5 object method vs the newer es6 style `thing: function(){}` vs `thing(){}` + some ember object magic thrown in there for good measure.